packages/generator/src/parser/parser-types-v4.ts

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+
+import {
+  EdmxMetadataBase,
+  EdmxEntityTypeBase,
+  EdmxEntitySetBase,
+  EdmxFunctionImportBase,
+  EdmxComplexTypeBase,
+  EdmxNamed
+} from './parser-types-common';
+
+export interface EdmxMetadata extends EdmxMetadataBase {
+  entitySets: EdmxEntitySet[];
+  entityTypes: EdmxEntityType[];
+  enumTypes: EdmxEnumType[];
```

Comment:
In the final `VdmServiceMetadata` created by the parse methods I do not see a representation of the enumTypes. Only EntitySets, functionImports and complexTypes where the Entity sets reference the entityTypes. Where do the enumTypes go?

Reply:
This is not implemented, I only added enum types, to make sure it does not fail (and initially to have a difference between v2 / v4

Reply:
They are not implemented yet. In the end I assume we will have a common representation, that just won't be used in the v2 case.